content/plugins/aria.md

```diff
@@ -28,7 +28,14 @@ The ARIA plugin uses ARIA version 0.1.1.
       
 
 # Terminology
-
+* **Service-Template** - A TOSCA equivalent to Cloudify's blueprint concept. 
+For additional info please refer the TOSCA [spec](http://docs.oasis-open.org/tosca/TOSCA/v1.0/os/TOSCA-v1.0-os.html#_Toc356403659).  
```

Comment:
Use this link (latest version)
http://docs.oasis-open.org/tosca/TOSCA/v1.0/TOSCA-v1.0.html